uaclient/cli.py

```diff
@@ -568,17 +537,24 @@ def action_enable(args, cfg, **kwargs):
     valid_services_names = entitlements.valid_services(allow_beta=args.beta)
     ret = True
 
-    for entitlement in entitlements_found:
+    for ent_name in entitlements_found:
         try:
-            ent_ret = _perform_enable(
-                entitlement,
-                cfg,
-                assume_yes=args.assume_yes,
-                allow_beta=args.beta,
+            ent_cls = entitlements.ENTITLEMENT_CLASS_BY_NAME[ent_name]
+            entitlement = ent_cls(
+                cfg, assume_yes=args.assume_yes, allow_beta=args.beta
             )
-
-            if not ent_ret and entitlement not in valid_services_names:
-                entitlements_not_found.append(entitlement)
+            ent_ret = entitlement.enable()
```

Comment:
Just to confirm, we are not passing `silent_if_enable` here anymore because  by default this attribute is false and we are not allowing user to change its value through CLI, right ?

Reply:
Correct! And this isn't a functional change from before.
Before, the silent param was being set to the default of False by the old _perform_enable, and now it is getting the default of false by ent.enable